### ISSUE TITLE
No match data

### ISSUE THREAD

**Issue by BadSpider1** [PhlexPlexico/G5API#247]
I have no idea which part of the automation actually causes the issue so i have chosen to create the issue here

Every match there are no data (kills MVP etc.)

"No stats are found for the current match."

**Comment by PhlexPlexico** [comment]
Hi,

Have you tried using the public panel to see if it's reproducible? What version of get5 are you running, and do you have steamworks installed?

**Comment by BadSpider1** [comment]
Yeah, i used the public version and only today moved to my own private instance
SteamWorks is installed and get5 version is [0.14.5](https://github.com/splewis/get5/releases/tag/v0.14.5)

**Comment by PhlexPlexico** [comment]
Get5 as of 0.14 is supported. Please use the latest 0.13 if you wish to use this API

**Comment by BadSpider1** [comment]
Right, so all i Gotta do is downgrade to 
[0.13.1](https://github.com/splewis/get5/releases/tag/v0.13.1) ?

**Comment by Bunker56** [comment]
get5_remote_backup_url, get5_demo_path, etc. - should it have any meaning or not, because when using 0.13 for some reason the demo is not recorded, but when using 0.10.5 everything is fine.

**Comment by BadSpider1** [comment]
I dont use upload demos anyways

**Comment by Bunker56** [comment]
Then everything will work, tested.

**Comment by PhlexPlexico** [comment]
There should be no difference and demo recording should work on the last version of 0.13 as well, just fyi. If you're having an issue with that @Bunker56, you should report it as a new issue with as much information as possible (logs, plugins being used, a way to reproduce). Last I checked with the latest G5WS plugin on 0.23, everything was working as intended.

Thanks!

**Comment by Bunker56** [comment]
> I checked with the latest G5WS plugin on 0.23

get5-6560baa40b?
